mava/environment_loop.py

```diff
@@ -575,28 +580,61 @@ def should_run_loop(eval_condtion: Tuple) -> bool:
                         "Running eval loop at executor step: "
                         + f"{self._last_evaluator_run_t}"
                     )
+
             return should_run_loop
 
         episode_count, step_count = 0, 0
 
-        # Currently, we only use intervals for eval loops.
-        environment_loop_schedule = (
-            self._executor._evaluator and self._executor._interval
-        )
-        if environment_loop_schedule:
-            eval_condition = check_count_condition(self._executor._interval)
+        # TODO (Ruan): Checking whether we are using Jax or TF here
+        # this should be removed once we deprecate TF.
```

Comment:
Removing TF will make the code much more readable

Reply:
Agreed! :muscle: